### PR TITLE
use linesIterator instead on lines in project/*.scala files

### PR DESCRIPTION
**Pull request by xuwei-k**

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -32,3 +32,5 @@ concurrentRestrictions in Global := Seq(
 )
 
 addSbtPlugin("pl.project13.scala" % "sbt-jmh" % "0.3.3")
+
+scalaVersion := "2.12.7"
```
Comment (xuwei-k) [inline project/plugins.sbt:36]: https://github.com/scala/scala/blob/c277d7d7ca79d0f66bb458bcfb03c060b77b9357/project/plugins.sbt#L2

We should add `scalaVersion` setting due to `"-Xfatal-warnings"`

Comment (lrytz) [reply project/plugins.sbt:36]: Why? Which Scala version does sbt use otherwise?

Comment (xuwei-k) [reply project/plugins.sbt:36]: sbt 1.2.3 use Scala 2.12.6
- https://github.com/scala/scala/blob/v2.12.6/src/library/scala/collection/immutable/StringLike.scala#L136-L137
- https://github.com/scala/scala/blob/v2.12.7/src/library/scala/collection/immutable/StringLike.scala#L136-L137
- https://github.com/scala/scala/commit/46db85852564f123697666da24ef216c5b2b29b3

Comment (lrytz) [reply project/plugins.sbt:36]: Thank you!

**Review by lrytz — APPROVED** [review]

**Pull request merged**